Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 The Service Manager Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: build test
+
+BINDIR ?= bin
+TEST_PROFILE ?= $(CURDIR)/profile.cov
+COVERAGE ?= $(CURDIR)/coverage.html
+PROJECT_PKG = github.com/Peripli/service-manager
+
+PLATFORM ?= linux
+ARCH     ?= amd64
+
+BUILD_LDFLAGS =
+
+#  GOFLAGS -  extra "go build" flags to use - e.g. -v   (for verbose)
+GO_BUILD = env CGO_ENABLED=0 GOOS=$(PLATFORM) GOARCH=$(ARCH) \
+           go build $(GOFLAGS) -ldflags '-s -w $(BUILD_LDFLAGS)'
+
+build: .init dep-vendor service-manager
+
+dep-check:
+	@which dep 2>/dev/null || go get -u -v github.com/golang/dep/cmd/dep
```

Comment:
Dep installation is better to be a prerequisite. 
Make check for dep and print message for missing tool instead installing it.